portal-api/src/v1/controllers/eligibilityCriteria.controller.js

```diff
@@ -50,14 +50,27 @@ exports.findAll = (req, res) =>
       res.status(200).send({
         count: eligibilityCriteria.length,
         eligibilityCriteria: sortedEligibilityCriteria,
-      })));
+      }),
+    ),
+  );
 
 exports.findOne = (req, res) => findOneEligibilityCriteria(Number(req.params.version), (eligibilityCriteria) => res.status(200).send(eligibilityCriteria));
 
+/**
+ * Finds the latest eligibility criteria document from the 'eligibilityCriteria' collection.
+ * EC is returned as an array for mapping.
+ * 
+ * @returns {Promise<Object>} The latest eligibility criteria document.
```

Comment:
Is this Promise<Object[]>?

Reply:
Good spot, nope just an object have updated the DocString for both.